lib/runners/processor/pmd_cpd.rb

```diff
@@ -71,6 +71,10 @@ class Processor::PmdCpd < Processor
 
     register_config_schema(name: :pmd_cpd, schema: Schema.runner_config)
 
+    def force_option_skip_lexical_errors
+      @force_option_skip_lexical_errors = true
+    end
```

Comment:
[suggestion] In this case, using `attr_writer` is natural:

```suggestion
    attr_writer :force_option_skip_lexical_errors
```

Reply:
Thank you for pointing out. It helps to reduce the number of lines.